crates/stages/src/stages/hashing_account.rs

```diff
@@ -63,20 +63,20 @@ impl AccountHashingStage {
     pub fn seed<DB: Database>(
         tx: &mut Transaction<'_, DB>,
         opts: SeedOpts,
-    ) -> Result<Vec<(Address, Account)>, StageError> {
+    ) -> Result<Vec<(reth_primitives::Address, reth_primitives::Account)>, StageError> {
```

Comment:
Revert this qualification

Reply:
It is used inside `test_utils` flag that is why it has `reth_primitives::` there